src/resources/nfts/simplehash/utils.ts

```diff
@@ -329,68 +336,65 @@ export function simpleHashNFTToInternalNFT(nft: ValidatedSimpleHashNFT): NFT {
     nft.image_properties?.mime_type === SVG_MIME_TYPE
   );
 
+  // filter out unsupported marketplaces
   const marketplaces = nft.collection.marketplace_pages
     .filter(
-      (marketplace: SimpleHashMarketplace) =>
-        // filter out unsupported marketplaces
-        !!getInternalMarketplaceIdFromSimpleHashMarketplaceId(
-          marketplace.marketplace_id
-        )
+      m =>
+        !!getInternalMarketplaceIdFromSimpleHashMarketplaceId(m.marketplace_id)
     )
-    .map(
-      (marketplace: SimpleHashMarketplace) =>
-        ({
-          collectionId: marketplace?.marketplace_collection_id,
-          collectionUrl: marketplace?.collection_url,
-          marketplaceId: getInternalMarketplaceIdFromSimpleHashMarketplaceId(
-            marketplace.marketplace_id
-          ),
-          name: marketplace?.marketplace_name,
-          nftUrl: marketplace?.nft_url,
-        } as NFTMarketplace)
-    );
+    .map((m: SimpleHashMarketplace) => {
+      const marketplace: NFTMarketplace = {
+        collectionId: m?.marketplace_collection_id,
+        collectionUrl: m?.collection_url,
+        marketplaceId: getInternalMarketplaceIdFromSimpleHashMarketplaceId(
+          m.marketplace_id
+        )!,
+        name: m?.marketplace_name,
+        nftUrl: m?.nft_url,
+      };
```

Comment:
So doing it this way makes it clear which fields we're overriding, and will ensure that new fields or other edits will for TS to validate the `marketplace` const here.